### DISCUSSION TITLE
Use/recreate Wowchemy widgets?

### DISCUSSION THREAD

**Discussion by noah-silver** [quarto-dev/quarto-cli#1563] [Q&A] [answered]
I have a website built on Wowchemy academic theme that I am trying to recreate in Quarto. I prefer Quarto for the file management and efficiency, but I prefer certain aspects of the Wowchemy academic theme aesthetically. In particular, I love the Wowchemy experience widget. I've found the source [css](https://github.com/wowchemy/wowchemy-hugo-themes/blob/e0ae7c979cda39124bd93b51868b58b70d5a1d70/wowchemy/assets/scss/wowchemy/widgets/_experience.scss) and [widget html](https://github.com/wowchemy/wowchemy-hugo-themes/blob/e0ae7c979cda39124bd93b51868b58b70d5a1d70/wowchemy/layouts/partials/widgets/experience.html) files, and I've used the css to customize my experience in Quarto, but I can't port the widget for use in Quarto. The closest I've come to the Wowchemy experience widget is to add my experience in a Quarto listing.

**Comment by cderv** [comment, marked as answer]
I wonder if you could not use the Custom listing feature to recreate the page that you are looking for.
https://quarto.org/docs/websites/website-listings-custom.html

Did you try this approach yet ?

**Reply by noah-silver** [reply]
This seems like a great option, thanks!

**Reply by cderv** [reply]
This should be a good way to achieve what you need to do. Please report back if something is not working as expect ! Thanks
